tox.ini

```diff
@@ -4,6 +4,8 @@ envlist=py,linter,mypy,docs,coverage-report
 
 [testenv]
 extras = test
+deps =
+    hatchling
```

Comment:
I'm not sure we need this, but I expect we will remove this file completely in a follow-up PR so we shouldn't block on that.

Reply:
Right, without this couple of tests are failing with below error
```
pyproject_hooks._impl.BackendUnavailable: Cannot import 'hatchling.build'
```